rfcs/interop_2023.md

```diff
@@ -0,0 +1,100 @@
+# RFC 116: Interop 2023
+
+## Summary
+
+Interop 2023 is an effort to improve interoperability in areas believed to be important to web developers and users. This is the next iteration of [Interop 2022](./interop_2022.md), an effort which is still ongoing at the time of this RFC.
+
+The focus areas will be selected by a proposals period followed by a proposals review and selection process. The [Interop team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will manage this process and make decisions based on consensus, see [governance](#governance).
+
+There will be a dashboard at https://wpt.fyi/interop-2023.
+
+## Details
+
+The Interop 2022 [repo](https://github.com/web-platform-tests/interop-2022) and [team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will be renamed to omit the year and used for this effort as well.
```

Comment:
We should figure out how to handle the existing issues / proposals in this framework. Options are:
* Rename labels to include the year
* Add milestones for each year and add the issues to those milestones.

A concrete use case here is that there are non-accepted proposals from 2022 that people want to audit for inclusion in 20223.

Reply:
> Rename labels to include the year

This seems somewhat annoying further in the future; one could easily imagine having to search for `-label:accepted:2022 -label:accepted-2023` etc. etc.

Reply:
How about `proposal 2022` and `proposal 2023` but no other labels (like `accepted`) get dates?

Reply:
I'll add this:

> Proposals from different years will be distinguished by labels or milestones.